sqrl-planner/src/main/java/com/datasqrl/config/RootPath.java

```diff
@@ -18,12 +18,8 @@
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
 import java.nio.file.Path;
-import lombok.Getter;
 
-@Getter
-public class RootPath {
-
-  private final Path rootDir;
+public record RootPath(Path rootDir) {
 
   @Inject
   public RootPath(@Named("rootDir") Path rootDir) {
```

Comment:
wouldn't be better to annotated the field on record with `@Named("rootDir")` 
and use `@AllArgConstructor` to generate constructo with `@Inject`

DI with java records look odd

Reply:
Lombok ctor annotations cannot be put on records.